### DISCUSSION TITLE
Docusaurus + pdoc question

### DISCUSSION THREAD

**Discussion by Valahaar** [h2oai/wave#1134] [Q&A] [answered]
Hey!

Really cool project you got going on here :)

I stumbled upon it trying to make pdoc work with docusaurus, and I see that you managed to do it perfectly (great docs btw!), so hopefully you can help me. Sorry if this is not related to the project itself, I've tried to contact you through twitter but your handle(s) don't allow DMs.

I'm stuck at a point where Docusaurus won't convert markdown embedded within HTML (like in your case, in the mako template you have `<div className="api">## Classes</div>` which is correctly rendered in its `<h2>` form), and I can't figure out how to make it work like you guys did. 

Again, sorry if this is completely unrelated to the project! If there's another place where we could discuss this, please do let me know.

Thanks,
Niccolò

**Comment by mturoci** [comment]
Hi @Valahaar, this question is not Wave-related and more suited for StackOverflow where more people can benefit from the answer (more discoverable than this repo).

>like in your case, in the mako template you have \<div className="api">## Classes</div> which is correctly rendered in its \<h2> form

I was not able to find this line, can you link it?

Generally speaking, I don't think what you are trying to achieve is possible since markdown also supports HTML, meaning the `## Classes` within should be correctly interpreted as a string, rather than parsed as a markdown. I suggest using
```html
<div className="API">
<h2>Classes</h2>
</div>
```
which gives you the same effect.

**Comment by Valahaar** [comment, marked as answer]
If I keep an empty line between markdown and HTML it works.

Basically,
```html
<div>

### Subclasses

</div>
```
works but
```html
<div>
### Subclasses
</div>
```
doesn't.

You can close this, sorry again for the OT question!
